### PR TITLE
feature/docker-salt-minion

### PR DESCRIPTION
**Pull request by Andreichenko**
added

### CODE REVIEW

**Comment by Andreichenko** [comment]
please review @freideveloper

File: salt-minion/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,45 @@
-FROM ubuntu:trusty
+FROM centos:7
+
 MAINTAINER Sascha F. Andreichenko <andreichenko@web.de>
-RUN apt-get update && apt-get install -y salt-minion curl
-COPY setup.sh /opt/setup.sh
-ENTRYPOINT ["sh","/opt/setup.sh"]
-CMD []
```
Comment (freideveloper) [inline salt-minion/Dockerfile:6]: OK

File: salt-minion/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,45 @@
-FROM ubuntu:trusty
```
Comment (freideveloper) [inline salt-minion/Dockerfile:1]: OK

**Review by freideveloper — APPROVED** [review]
Need to test

**Comment by Andreichenko** [comment]
> Need to test

agree

**Pull request merged**